### PR TITLE
fall quarter updates

### PR DESCRIPTION
**Pull request by limpa105**
List of Updates 
- FMCAD paper news 
- Tutorial FM news 
- Graduated Ph.D. Students 
- Graduated Master Students
- Rachel headshot 
- New Sponsors
- Founding Sponsors
Screenshot 2024-09-27 at 2 13 18 PM

List of Remaining Updates
- Classes page (waiting for responses)
- Rotating students (need to figure out how to add new class)
- Bachelor students

### CODE REVIEW

File: _data/news_items.yaml [hunk 1]
```diff
@@ -1,3 +1,24 @@
+- title: "SMT tutorial paper has been selected as distinguished tutorial at FM 2024"
+  urls:
+    - url: "https://hanielbarbosa.com/papers/fm2024.pdf"
+      label: "Satisfiability Modulo Theories: A Beginner’s Tutorial"
+  excerpt: "<b> Satisfiability Modulo Theories: A Beginner's Tutorial </b> by Clark Barrett, Cesare Tinelli, Haniel Barbosa, Aina Niemetz,
+   Mathias Preiner, Andrew Reynolds and Yoni Zohar has been selected as a distinguished tutorial at  the <i> 26th International Symposium on Formal Methods </i> located in Milan, Italy!"
```
Comment (barrettcw) [inline _data/news_items.yaml:6]: I am pro-Oxford comma

File: _data/news_items.yaml [hunk 2]
```diff
@@ -1,3 +1,24 @@
...
+
+- title: "4 papers from CENTAUR have been accepted to FMCAD 2024!"
+  urls:
+    - url: "https://arxiv.org/abs/2405.14058"
+      label: "Formally Verifying Deep Reinforcement Learning Controllers with Lyapunov Barrier Certificates"
+    - url:  "https://arxiv.org/abs/2405.06127"
+      label: "Efficiently Synthesizing Lowest Cost Rewrite Rules for Instruction Selection"
+    - url: "https://repositum.tuwien.at/bitstream/20.500.12708/200789/1/Cleaveland-2024-Memory%20Consistency%20Model-Aware%20Cache%20Coherence%20for%20Hetero...-vor.pdf"
+      label: Memory Consistency Model-Aware Cache Coherence for Heterogeneous Hardware
+  excerpt: " Papers by Udayan Mandal, Ross Daly and Rachel Cleaveland as well as a tool paper 'SMT-D: New Strategies for Portfolio-Based SMT Solving' have been accepted to FMCAD2024. "
```
Comment (barrettcw) [inline _data/news_items.yaml:18]: Oxford comma :)

File: _data/people.yaml [hunk 3]
```diff
@@ -75,6 +81,14 @@
   start: 2020
   end: 2023
 
+- name: Omri Isac
+  position: visiting
+  start: 2024
+  end: 2024
```
Comment (barrettcw) [inline _data/people.yaml:87]: Well, since today is his last day, we might as well not add him now,

File: affiliates_and_sponsors.md [hunk 4]
```diff
@@ -29,12 +18,30 @@ title: affiliates and sponsors
...
+</dl>
+<h4> Silver </h4>
+<dl>
+<dd> Adacore, Certora </dd>
+</dl>
+<h4> Additional </h4>
+<dl>
+<dd> Vmware </dd>
+</dl>
+
```
Comment (barrettcw) [inline affiliates_and_sponsors.md:43]: Let's just have a bulleted list without the categories (i.e., Platinum, Silver, etc.)

File: _data/people.yaml [hunk 5]
```diff
@@ -92,6 +106,7 @@
 - name: Rachel Cleaveland
   position: phd
   start: 2022
+  img: cleaveland.jpg
 
 - name: Caleb Donovick
   position: phd
```
Comment (barrettcw) [inline _data/people.yaml:112]: Caleb has also graduated

File: _data/people.yaml (hunk 3 above)
Comment (barrettcw) [reply _data/people.yaml:87]: I guess we can add him as alumni :)

File: _data/people.yaml [hunk 6]
```diff
@@ -162,17 +167,46 @@
...
 
 - name: Udayan Mandal
   position: master
   start: 2022
-
-# Bachelor students
-
-# Alumni
+  end: 2024
+  now: Atlassian
```
Comment (barrettcw) [inline _data/people.yaml:209]: We should also add Ben, Kaia, and Jibiana as alumni

**Review by barrettcw — CHANGES_REQUESTED** [review]
Looks great - thanks!  Just a few minor comments

File: _data/people.yaml [hunk 7]
```diff
@@ -162,17 +168,63 @@
...
+- name: Jibiana Jakpor
+  position: bachelor
+  start: 2024
+  end: 2024
+
+- name: Benjamin Przybocki
+  position: bachelor
+  start: 2023
+  end: 2024 
+  now: Oxford
```
Comment (barrettcw) [inline _data/people.yaml:187]: Ben is at Cambridge, not Oxford

File: _data/people.yaml [hunk 8]
```diff
@@ -162,17 +168,63 @@
...
+  end: 2024 
+
+
+- name: Ying Sheng
+  position: phd
+  img: sheng.jpg
+  website: https://profiles.stanford.edu/ying-sheng
+  start: 2018
+  end: 2024
+  now: UCLA
```
Comment (barrettcw) [inline _data/people.yaml:201]: So, Ying is currently at XAI, and will start at UCLA in a couple of years.  Maybe just put both?

File: _data/people.yaml [hunk 9]
```diff
@@ -162,17 +168,63 @@
...
+
+- name: Sophia Andrews
```
Comment (barrettcw) [inline _data/people.yaml:203]: Sophie not Sophia

**Review by barrettcw — CHANGES_REQUESTED** [review]
Almost there...

File: _data/people.yaml (hunk 9 above)
Comment (limpa105) [reply _data/people.yaml]: Thank you for catching these mistakes!

**Review by barrettcw — APPROVED** [review]
LGTM!

**Pull request merged**